index.html

```diff
@@ -40,14 +40,14 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
 
             <div class="form-group">
               <p>Which option best describes your current role?</p>
```

Comment:
-  Please add two spaces of indentation for your code to be easily read. 
You can set the tab button to do an X amount of spaces.

Reply:
- [ ] Great job doing the review but I think here it would be great if you add the reason because maybe it is obvious to you but not for the others ✅👌🏻👌🏻💯💯